libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccountManager.java

```diff
@@ -202,11 +189,10 @@ public Account getCurrentAccount() {
         	if (account != null) {
 
         		// Reads the user ID and org ID from account manager.
-                String passcodeHash = SalesforceSDKManager.getInstance().getPasscodeHash();
-				final String orgId = SalesforceSDKManager.decryptWithPasscode(accountManager.getUserData(account,
-                		AuthenticatorService.KEY_ORG_ID), passcodeHash);
-        		final String userId = SalesforceSDKManager.decryptWithPasscode(accountManager.getUserData(account,
-        				AuthenticatorService.KEY_USER_ID), passcodeHash);
+				final String orgId = SalesforceSDKManager.decrypt(accountManager.getUserData(account,
```

Comment:
Replaced all `decryptWithPasscode...` methods in this file with `decrypt...` and `encryptWithPasscode...` methods with `encrypt...`.